src/main/scala/chisel3/iotesters/DriverCompatibility.scala

```diff
@@ -10,6 +10,7 @@ import firrtl.options.Viewer.view
 import firrtl.options.phases.DeletedWrapper
 import firrtl.options.{Dependency, OptionsException, OptionsView, OutputAnnotationFileAnnotation, Phase, PhaseManager, StageError, Unserializable}
 import firrtl.stage.phases.DriverCompatibility.TopNameAnnotation
+import firrtl.stage.phases.DriverCompatibilityExtensions
```

Comment:
This is unfortunately needed. Provided by https://github.com/freechipsproject/firrtl-interpreter/pull/223